core-rust/mesh-api-server/src/mesh_api/handlers/network_options.rs

```diff
@@ -20,23 +20,17 @@ pub(crate) async fn handle_network_options(
             metadata: None,
         }),
         allow: Box::new(models::Allow {
-            operation_statuses: vec![
-                models::OperationStatus::new("SuccessStatus".to_string(), true),
-                models::OperationStatus::new("FailureStatus".to_string(), false),
-            ],
-            // TODO::MESH Add enum with operation types
-            // see comment https://github.com/radixdlt/babylon-node/pull/1013#discussion_r1830848173
-            operation_types: vec![
-                "Withdraw".to_string(),
-                "Deposit".to_string(),
-                "Mint".to_string(),
-                "Burn".to_string(),
-                "LockFee".to_string(),
-            ],
+            operation_statuses: MeshApiOperationStatus::iter()
+                .map(|s| models::OperationStatus::new(s.to_string(), s.into()))
```

Comment:
MINOR: an `Into<bool>` is a little indirect for my liking - it's not clear what the bool means (success or failure?).

Perhaps there should just be an `From< MeshApiOperationStatus> for models::OperationStatus` implementation?

Reply:
Good idea

Reply:
Done